clarify which credential format each example uses (#333)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

<{{examples/authorization_details.json}}

> The following is a non-normative example of an `authorization_details` object with a `format` parameter of the Credential format `vc+sd-jwt`:

Author: ```suggestion
The following is a non-normative example of an `authorization_details` object requesting a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```

<{{examples/authorization_details.json}}

> The following is a non-normative example of an `authorization_details` object requesting a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:

Reviewer: ```suggestion
The following is a non-normative example of an `authorization_details` object requesting a Credential in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```

> The following is a non-normative example of an `authorization_details` object requesting a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:

Reviewer 2: I think this reads more naturally without the second 'an' (hopefully I'm not contradicting other people's suggestions here...):

```suggestion
The following is a non-normative example of an `authorization_details` object requesting a Credential in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```

* `credential_requests`: REQUIRED. Array that contains Credential Request objects, as defined in (#credential-request).

> Below is a non-normative example of a Batch Credential Request for two Credentials in an [@VC_DATA] format and [@ISO.18013-5] format:

Reviewer 3: I find this sentence hard to read, suggest the following

```suggestion
Below is a non-normative example of a Batch Credential Request for two Credentials; one in [@VC_DATA] format and another in [@ISO.18013-5] format:
```

Author: for the future guidance.. for editorial suggestions, would really appreciate approval conditional to suggested changes to being accepted. if I agree with the changes, I can just accept them and merge a PR without asking to re-review, and if i do not agree, I will reach back to you to clarify until we reach agreement.

> Below is a non-normative example of a Batch Credential Request for two Credentials in an IETF SD-JWT VC [@VC_DATA] format and [@ISO.18013-5] format:

Reviewer 3: ```suggestion
Below is a non-normative example of a Batch Credential Request for two Credentials in an IETF SD-JWT VC [@I-D.ietf-oauth-sd-jwt-vc] format and [@ISO.18013-5] format:
```

  * `proofs`: OPTIONAL. See (#credential-request).
* `credential_response_encryption`: OPTIONAL. Object containing information for encrypting the Batch Credential Response. It contains the same parameters as defined in #{credential-request}. If this request element is not present, the corresponding Batch Credential Response returned is not encrypted.

Below is a non-normative example of a Batch Credential Request requesting four Credentials:

> * Two Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format;

Reviewer: ```suggestion
* Two Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format;
```

> * Two Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format;

Reviewer 2: Same:

```suggestion
* Two Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format;
```

> * Two Credentials for the same Credential Configuration but with different Credential Dataset in [@VC_DATA] format.

Author: @paulbastian can you please re-review if this new text makes sense

Reviewer 4: Makes sense

> * Two Credentials for the same Credential Configuration but with different Credential Dataset in [@VC_DATA] format.

Reviewer: ```suggestion
* Two Credentials for the same Credential Configuration but with different Credential Datasets in the [@VC_DATA] format.
```

  * `proofs`: OPTIONAL. See (#credential-request).
* `credential_response_encryption`: OPTIONAL. Object containing information for encrypting the Batch Credential Response. It contains the same parameters as defined in #{credential-request}. If this request element is not present, the corresponding Batch Credential Response returned is not encrypted.

> Below is a non-normative example of a Batch Credential Request for two Credentials; one in [@VC_DATA] format and another in [@ISO.18013-5] format:

Author: ```suggestion
Below is a non-normative example of a Batch Credential Request for four Credentials, all in [@VC_DATA] format:
* 2 Credentials for the same Credential Configuration and Credential Dataset but with different cryptographic binding keys;
* 2 Credentials for the same Credential Configuration but with different Credential Dataset.
```

The Authorization Server MUST be able to determine from the Issuer metadata what claims are disclosed by the requested Credentials to be able to render meaningful End-User consent.

> The following is a non-normative example of Credential Issuer metadata with Credential format `vc+sd-jwt`:

Reviewer: ```suggestion
The following is a non-normative example of Credential Issuer metadata with the Credential format `vc+sd-jwt`:
```

The Authorization Server MUST be able to determine from the Issuer metadata what claims are disclosed by the requested Credentials to be able to render meaningful End-User consent.

> The following is a non-normative example of Credential Issuer metadata of a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:

Reviewer 2: ```suggestion
The following is a non-normative example of Credential Issuer metadata of a Credential in the IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```

The Authorization Server MUST be able to determine from the Issuer metadata what claims are disclosed by the requested Credentials to be able to render meaningful End-User consent.

> The following is a non-normative example of Credential Issuer metadata with the Credential format `vc+sd-jwt`:

Author: ```suggestion
The following is a non-normative example of Credential Issuer metadata of a Credential in an IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc] format:
```
